crates/sui-json-rpc/src/coin_api.rs

```diff
@@ -1396,13 +1431,50 @@ mod tests {
             expected.assert_eq(&result.value.to_string());
         }
 
+        #[tokio::test]
```

Comment:
Is this test function name the same as the one above, or am I just tripping?

Reply:
not tripping - they are, but under different modules